documentation/crds_users_guide/source/library_use.rst

```diff
@@ -147,7 +147,7 @@ reference type:
         filling in appropriate reference type keywords.
     
         Define best references using either .pmap `context` or the default
-        CRDS operational context if context=None.
+        CRDS latest context if context=None.
```

Comment:
Just remove the word "latest": "Define best references using either .pmap `context` or the default CRDS context if context=None."